tests/integration/config/plugins/integration-tests/data/tests/tests_common.txt

```diff
@@ -539,6 +539,120 @@ test "Load First Savegame"
 			key l
 
 
+test "Create New Pilot"
+	status partial
+	description "Helper function to create a new pilot."
+	sequence
+		input
+			key l
+		# new pilot
+		input
+			key n
+		# start
+		input
+			key s
+		# first name: a
+		input
+			key a
+		input
+			key "return"
+		# last name: b
+		input
+			key b
+		input
+			key "return"
+		# skip text
+		input
+			key "return"
+		# go to the main landing panel
+		input
+			key l
+
+
+test "Buy First Ship"
+	status partial
+	description "Helper function to buy the first ship of the shipyard, starting from the planet panel."
+	sequence
+		input
+			key s
+		# buy first ship
+		input
+			key "down"
+		input
+			key b
+		# confirm
+		input
+			key "return"
+		# back to landing
+		input
+			key l
+
+
+test "Wait 55 steps"
+	status partial
+	description "Wait 55 steps for something that requires some time, like launching or loading the game."
+	sequence
+		apply
+			"test: wait 55" = 55
+		label wait_load
+		apply
+			"test: wait 55" -= 1
+		input
+		branch wait_load
+			"test: wait 55" > 0
+
+
+test "Wait 10 steps"
+	status partial
+	description "Wait 10 steps to give time for the game to get the newest conditions. 5 steps are just enough but may as well be sure."
```

Comment:
A test that is called `Wait 10 steps` should just wait for 10 steps, without an opinion on if 10 steps are enough. I suggest to create a more neutral description, or to rename the test to "Wait for conditions to be propagated".

Reply:
```suggestion
	description "Wait 10 steps, for instance to give time for the game to propagate conditions, or load something."
```